README.md

```diff
@@ -1 +1,42 @@
-# pixelbin-figma-plugin
\ No newline at end of file
+# PixelBin.io Figma Plugin
```

Comment:
Add the following instructions :- 

1. how to add new transformation to either AI drawer or Basic Drawer - incase automatic mention the same 
2. What are some custom code you have written to make box selection etc work, point to those

Reply:
Mention project structure, pointing to correct files in case of different use cases, the main ones, like download, upload and transform